dashboard/src/components/forms/attributes/AgeInput.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useContext, useEffect } from "react";
+import { Box, HStack, FormControl, FormLabel, Input } from "@chakra-ui/react";
+
+import configData from "../../../config/app_config.json";
+import { HouseholdContext } from "../../../contexts/HouseholdContext";
+import { ErrorMessage } from "./validation/ErrorMessage";
+
+export const AgeInput = ({
+  personName,
+  mustInput,
+}: {
+  personName: string;
+  mustInput: boolean;
+}) => {
+  const { household, setHousehold } = useContext(HouseholdContext);
+  const [age, setAge] = useState("");
+
+  const handleAgeChange = (event: React.ChangeEvent<HTMLInputElement>) => setAge(event.currentTarget.value);
+
+  useEffect(() => {
+    let birthday;
```

Comment:
To prevent erroneous operation, please prohibit the input of negative values (display and set 0).  
(reference: `dashboard/src/components/forms/attributes/ChildrenNum.tsx`)

Reply:
ありがとうございます！直しました！